sceptre/bridge/config/develop/BridgeServer2-vpc.yaml

```diff
@@ -7,3 +7,4 @@ dependencies:
 parameters:
   VpcName: BridgeServer2-develop-vpc
   VpcSubnetPrefix: "172.48"
+  IncludeBastianSecurityGroup: "false"
```

Comment:
i would double check to make sure bridge isn't using these SG before merging

Reply:
I executed aws --region us-east-1 --profile <profile> ec2 describe-network-interfaces --filters Name=group-id,Values=<bastionSGid> in bridgedev/bridgeprod and it returned nothing. Found a couple more groups in the process (bridge-aux was also using the vpc template, and bridge was using its own template to setup the SG), I added them to this PR.